website/docs/guides/adding-authentication-with-auth0.mdx

```diff
@@ -0,0 +1,286 @@
+---
+title: Adding Authentication with Auth0
+sidebar_label: Adding Authentication with Auth0
+---
```

Comment:
Maybe add a note that Auth0 has many ways of integrating their client SDK, and in this example, you are using it from a CDN for simplicity. 
And you can link to different integrations (for example https://github.com/auth0/nextjs-auth0 )

Reply:
https://github.com/n1ru4l/envelop/blob/ad6450ee2aa9de5f4e3e74ee5e156f000a822070/website/docs/guides/adding-authentication-with-auth0.mdx#L170-L171